File: graphql-endpoint/src/main/java/nl/openweb/graphql_endpoint/model/Account.java
package nl.openweb.graphql_endpoint.model;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.util.UUID;

@Entity
@Table(name = "account")
@Data
public class Account {
    @Id
    private String username;
    private String password;
    private UUID uuid;
}

File: graphql-endpoint/src/main/kotlin/nl/openweb/graphql_endpoint/model/Account.kt
package nl.openweb.graphql_endpoint.model

import java.util.*
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "account")
class Account(
    @Id
    var username: String,
    var password: String,
    var uuid: UUID
)